client/src/modules/dashboard/Sponsors/pages/SponsorsPage.tsx

```diff
@@ -42,15 +47,16 @@ export const SponsorsPage: NextPage = () => {
                   ),
                   type: (sponsor) => sponsor.type,
                   website: (sponsor) => sponsor.website,
-                  action: (sponsor) => (
-                    <LinkButton
-                      colorScheme="blue"
-                      size="xs"
-                      href={`/dashboard/sponsors/${sponsor.id}/edit`}
-                    >
-                      Edit
-                    </LinkButton>
-                  ),
+                  action: (sponsor) =>
+                    useCheckPermission(Permission.SponsorManage) && (
+                      <LinkButton
```

Comment:
there is a similar button for mobile, we have to add the permission for, as well

Reply:
nvm. 🤣